cactusbot/services/beam/liveloading.py

```diff
@@ -13,7 +13,7 @@
 class BeamLiveloading(WebSocket):
```

Comment:
Same here - should rename to `constellation.py`, with class name `BeamConstellation`.